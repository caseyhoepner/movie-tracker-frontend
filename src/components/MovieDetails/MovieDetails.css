.details-container {
  width: 70%;
  margin: auto;
}

.backdrop {
  position: absolute;
  height: 100vh;
  z-index: -1;
  background: 
}

.back-btn {

}

.text-container {
  display: flex;
  flex-direction: column;
}

.title {

}

.overview {

}