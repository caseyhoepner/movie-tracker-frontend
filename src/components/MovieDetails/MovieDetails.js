import React from 'react';
import './MovieDetails.css';
import { Link } 'from react-router-dom';

const MovieDetails = (props) => {
  return (
    <div>
      <Link to="/">Back</Link>
      <h1>{ name }</h1>
      <img src={ poster_path } alt={`Poster of Wes Anderson's hit movie: "${name}"`}/>
      <p>{ overview }</p>
    </div>
  )
}

export default MovieDetails;