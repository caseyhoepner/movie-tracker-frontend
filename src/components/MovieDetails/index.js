import React from 'react';
import './MovieDetails.css';
import { Link } from 'react-router-dom';

const MovieDetails = (props) => {
  const { title, poster_path, overview, backdrop_path } = props;
  return (
    <div className='details-container'>
      <Link 
        className='back-btn' 
        to="/">Back</Link>
      <img 
        src={`http://image.tmdb.org/t/p/w342//${poster_path}`} 
        alt={`Poster of Wes Anderson's hit movie: "${title}"`}/>
      <div text-container>
        <h1 className='title'>{ title }</h1>
        <p className='overview'>{ overview }</p>
      </div>
    </div>
  )
}

export default MovieDetails;