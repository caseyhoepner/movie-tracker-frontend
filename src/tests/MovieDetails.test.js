import React from 'react';
import { shallow } from 'enzyme';
import MovieDetails from '../components/MovieDetails';

describe('MovieDetails', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<MovieDetails />);
  })

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot()
  })
})