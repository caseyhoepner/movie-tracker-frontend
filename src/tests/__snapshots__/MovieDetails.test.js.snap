// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieDetails should match the snapshot 1`] = `
<div
  className="details-container"
>
  <Link
    className="back-btn"
    replace={false}
    to="/"
  >
    Back
  </Link>
  <img
    alt="Poster of Wes Anderson's hit movie: \\"undefined\\""
    src="http://image.tmdb.org/t/p/w342//undefined"
  />
  <div
    text-container={true}
  >
    <h1
      className="title"
    />
    <p
      className="overview"
    />
  </div>
</div>
`;
